src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -1060,6 +1068,24 @@ export default abstract class ComparisonStore
         },
     });
 
+    @computed get genesWithMaxFrequency(): AlterationEnrichmentRow[] {
```

Comment:
This is taken from another incomplete PR, but I think the function name is not ideal. We are returning an array of `AlterationEnrichmentRow` and calling it `genesWithMaxFrequency`